concepticondata/references/references.bib

```diff
@@ -4266,5 +4266,13 @@ @Article{Nordenskioeld1905
   Number                              = {3},
   Journal                             = {Ymer},
   Title                               = {Beiträge zur Kenntnis einiger Indianerstämme des Rio Madre de Dios-gebietes [Contributions to the Knowledge about some Indian tribes of the area of the Rio Madre de Dios]},
-  Year                                = {1905},
+  Year                                = {1905}
+}
+
+@Book{RonaTas2011,
+  Author                              = {Róna-Tas, András, and Berta, Árpád},
+  Title                               = {West Old Turkic. Turkic loanwords in Hungarian},
```

Comment:
Most citation styles separate title and subtitle by colon, not by a dot

Reply:
Ah, according to this we use dots https://github.com/concepticon/concepticon-data/pull/1240#discussion_r991811756 but if you say it's more common with a comma, I'll go for a comma then

Reply:
I didn't see that comment, but then just keep the dot :)